bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/AngularVelocity.scala

```diff
@@ -0,0 +1,63 @@
+package edu.gemini.spModel.core
+
+import scalaz._, Scalaz._
+
+/** An angular velocity in [mas/y]  */
+case class AngularVelocity(masPerYear: Double) extends AnyVal with java.io.Serializable {
+
+  /**
+   * This `AngularVelocity` in degrees/year, it can be positive or negative
+   * @group Conversions
+   */
+  def toDegreesPerYear: Double = masPerYear / AngularVelocity.MilliArcSecsInADegree
+
+}
+
+/** An Angular Velocity in Right Ascension */
+case class RightAscensionAngularVelocity(velocity: AngularVelocity)
+
+/** An Angular Velocity in Declination */
+case class DeclinationAngularVelocity(velocity: AngularVelocity)
+
+object AngularVelocity {
+  val MilliArcSecsInADegree:Double = 1296000000
+
+  val Zero = AngularVelocity(0.0)
+
+  /** @group Typeclass Instances */
+  implicit val order: Order[AngularVelocity] =
+    Order.orderBy(_.masPerYear)
+
+  /** @group Typeclass Instances */
+  implicit val ordering: scala.Ordering[AngularVelocity] =
+    scala.Ordering.by(_.masPerYear)
+
+  /** @group Typeclass Instances */
+  implicit val equal: Equal[AngularVelocity] =
+    Equal.equalA[AngularVelocity]
+  
+  /** @group Typeclass Instances */
+  implicit val monoid: Monoid[AngularVelocity] =
+    Monoid.instance((a, b) => AngularVelocity(a.masPerYear + b.masPerYear), Zero)
```

Comment:
Any use of methods defined on `Any` (including `toString`) can be problematic when reasoning about types because it violates a principle called [**parametricity**](http://ttic.uchicago.edu/~dreyer/course/papers/wadler.pdf). So `Show` gives us a way of constraining a type parameter to only those types that explicitly provide a string representation. In practice I don't think it's very useful; there are no constraints on what `Show` can do so it doesn't really mean anything.

The issue of local/alternative instances is kind of thorny because it violates *another* principle called **coherence**, which requires that typeclass instances be unique. In general this is important, but using a local instance in a way that is not observable (like sorting a list using a non-default `Order`) is probably ok. 

But in any case I don't see much reason to define `Show` instances until we hit a point where we need them, because it may never happen.